Łoń Zofia/PD_6.ipynb

```diff
@@ -0,0 +1,95 @@
+{
```

Comment:
<p>Bardzo dobrze, fajnie by było wyróżnić jakimś kolorem średnią UE ze względu na to, że ten słupek prezentuje trochę inną informację niż pozostałe. </p><p> </p><p><code>5 punktów</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-ExploratoryDataAnalysis/pull/169/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>